src/tools/writeConfig.js

```diff
@@ -15,16 +15,19 @@ const writeConfig = () => {
 
         Object.keys(file).forEach((key) => {
           // If the key already exists in the project config, use it
-          if (projectConfig[key]) {
-            if (Array.isArray(projectConfig[key]) && Array.isArray(file[key])) {
-              projectConfig[key] = projectConfig[key].concat(file[key]);
-            } else {
-              projectConfig[key] = Object.assign(projectConfig[key], file[key]);
-            }
-          } else {
-            // Otherwise, set it
-            projectConfig[key] = file[key];
-          }
+         let configValue = projectConfig[key];
```

Comment:
Can you check your editorconfig/eslint settings here? Should match indentation style for the rest of the project